third_party/github.com/bazelbuild/rules_typescript/devserver/runfiles/runfiles.go

```diff
@@ -1,10 +1,34 @@
 // Package runfiles that provides utility helpers for resolving Bazel runfiles within Go.
 package runfiles
 
-import "github.com/bazelbuild/rules_go/go/tools/bazel"
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+)
 
 // Runfile returns the base directory to the bazel runfiles
 func Runfile(_base string, manifestPath string) (string, error) {
```

Comment:
oh wow! I've been struggling to remove our rules_go patch on their runfiles library for a year. This is great!

Reply:
It might need a little more testing, but lets see if I get to test it over the holydays